runtime/runtime-params-estimator/estimator-warehouse/src/import.rs

```diff
@@ -0,0 +1,172 @@
+use std::io::prelude::*;
+
+use anyhow::Context;
+
+use clap::Parser;
+use serde::Deserialize;
+use std::time::Duration;
+
+use crate::db::{EstimationRow, DB};
+
+/// Additional information required for import
+#[derive(Debug, Parser)]
+pub(crate) struct ImportConfig {
+    /// Required for importing estimation results, which source code commit it
+    /// should be associated with.
+    #[clap(long)]
+    pub commit_hash: Option<String>,
```

Comment:
Feels like we can shell-out to git from this binary to get the hash?

Reply:
For the daily job in buildkite, yes. But there are other cases to consider, where the binary can be from a different repository than the estimator. This has been the standard case while developing this (estimating master branch while working on my own branch).
Another case, at some point in the future we might figure out that feature X we just merged requires us to change some estimation functions. (E.g. new cache invalidates old estimation.) At that point, we want to compare "new estimation on new code" vs "new estimation on old code". It can always be done with some git history juggling but I would prefer my current solution.